IFC4x3/PropertyConstants/i/INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="INVERT_SEGMENT_WITH_SHOTCRETE_0ioCsK3H1CjgP5V2Oh9RYy" Name="INVERT_SEGMENT_WITH_SHOTCRETE" UniqueId="2cc8cd94-0d10-4cb6-a645-7c262b25b8bc" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)